src/app/lib/config/services/persian.js

```diff
@@ -52,49 +52,50 @@ const persian = {
     },
     consentBanner: {
       privacy: {
-        title: "We've updated our Privacy and Cookies Policy",
+        title: 'ما سیاست‌های حفظ حریم خصوصی و کوکی‌های خود را به روز کرده‌ایم',
         description: {
           uk: {
             first:
-              "We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data.",
+              'ما تغییرات مهمی در سیاست‌های حفظ حریم خصوصی و کوکی‌هایمان ایجاد کرده‌ایم و می‌خواهیم شما بدانید این تغییرات برای شما و اطلاعات مربوط به شما به چه معنی است',
             linkText: null,
             last: null,
             linkUrl: null,
           },
           international: {
             first:
-              "We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data.",
+              'ما تغییرات مهمی در سیاست‌های حفظ حریم خصوصی و کوکی‌هایمان ایجاد کرده‌ایم و می‌خواهیم شما بدانید این تغییرات برای شما و اطلاعات مربوط به شما به چه معنی است',
             linkText: null,
             last: null,
             linkUrl: null,
           },
         },
-        accept: 'OK',
-        reject: "Find out what's changed",
+        accept: 'تایید',
+        reject: 'ببنید چه تغییراتی ایجاد شده است',
         rejectUrl: 'https://www.bbc.co.uk/usingthebbc/your-data-matters',
       },
       cookie: {
-        title: 'Let us know you agree to cookies',
+        title: 'آیا با دریافت کوکی‌های ما موافقید؟',
         description: {
           uk: {
-            first: 'We use ',
-            linkText: 'cookies',
+            first: ' ما برای بهبود استفاده شما از خدمات بی‌بی‌سی از',
+            linkText: 'کوکی',
```

Comment:
The link text 'cookie' merges into the words following and preceding it, due to the lack of spaces
```suggestion
            linkText: ' کوکی ',
```

<img width="191" alt="Screen Shot cookie banner" src="https://user-images.githubusercontent.com/3028997/59248647-f345ff80-8c1a-11e9-9880-cf4ac649dfd6.png">

Reply:
Moving the space from the left end of 'first' to the right end of 'first', and the opposite for 'last' seems to have fixed this.

<img width="81" alt="Screen Shot 2019-06-11 at 12 13 23" src="https://user-images.githubusercontent.com/8124899/59267547-73cc2680-8c42-11e9-977c-927cd88723b5.png">